src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+                <tr>
+                    <td>Allowed Error</td>
+                    <td>Vertical distance threshold (must be &gt; 0). All points with their vertical distance &gt; threshold are declared as significant points and saved.</td>
+                </tr>
+                <tr>
+                    <td>Estimate Error</td>
+                    <td>If EstimateError &gt; 0 and the new point deviated from estimated value &gt; EstimateError, we will reduce/save the curve.</td>
+                </tr>
+                <tr>
+                    <td>Smoothing Coefficient</td>
+                    <td>Applies a low pass filter to the data.
+                        <br></br>
+                        0 = No filtering
+                        <br></br>
+                        1-254 = Smoothing coefficient magnitude
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Type 12 Logging With Timestamp</h4>
+        <p>This message can be used to send status data with a timestamp. Possible use cases:</p>
+        <ol>
+            <li>Store data in the IOX while the GO device is asleep and send all data after waking up</li>
+            <li>
+                Run the curve logging algorithm in the IOX and send those points to be transmitted to MyGeotab. 
+                <br></br>
+                Additional information about curve logging can be found here: <a href="https://github.com/Geotab/curve" target="_blank" rel="noreferrer">Curve Logging</a>
+            </li>
+        </ol>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Curve Logging</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>Status Data ID</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>Data Length</td>
+                </tr>
+                <tr>
+                    <td>3-6</td>
+                    <td>Data (unsigned 32bit)</td>
+                </tr>
+                <tr>
+                    <td>7</td>
+                    <td>Timestamp (# secs since 2002-01-01)</td>
+                </tr>
+                <tr>
+                    <td>8</td>
+                    <td>Milliseconds</td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Type 13 Protobuf Data</h4>
+        <p>Supported from Add-On protocol version 1.2.</p>
+        <p>This message allows an IOX to send a protobuf-encoded payload to the GO device. It supports a publish/subscribe model of vehicle status information. The GO device responds with GO Multi-Frame Data (0x27) - Type 13. <a href="https://github.com/Geotab/android-external-device-example/blob/master/app/src/main/proto/iox_messaging.proto" target="_blank" rel="noreferrer">Protobuf Schema</a>. The currently supported topics are:</p>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on